GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -1100,17 +1095,18 @@ private String buildOptionString(List<Map<String, String>> optionNodes) {
             }
         }
         if (optionString.length() > 0) {
-            // Remove the first |
+            // Remove the first "|"
             optionString.deleteCharAt(0);
         }
         return optionString.toString();
     }
 
     /*
-     * Creates list of option values. The first value is always the pipe-separated format Depending
-     * on the useQuestionId parameter, each option is given its own column A 0 or 1 denotes if that
-     * option was selected or not if the AllowOther flag is true, a column is created for the Other
-     * option. We first try to match on text, and if that fails, we try to match on code. This
+     * Creates list of option values. The value is always the pipe-separated format.
+     * Depending on the useQuestionId variable, each option is given its own column.
```

Comment:
wasn't useQuestionId renamed to splitIntoColumns

Reply:
It was. The replace did not reach into the comment.